src/java.base/share/classes/java/util/AbstractMap.java

```diff
@@ -741,6 +744,8 @@ public String toString() {
      * {@link Map#entry Map::entry} and {@link Map.Entry#copyOf Map.Entry::copyOf}
      * methods.
      *
+     * @param <K> the type of keys maintained
+     * @param <V> the type of mapped values
```

Comment:
Please update to match Map.Entry, which says simply "the type of key" and "the type of the value"